### PR TITLE
wallace/oc/0/dev

### PR DESCRIPTION
**Pull request by w12l3-c**
I made the interface with everything and the models an stuff, also include a README.md. The interface is in main.py of OC

### CODE REVIEW

File: models/outside-cattle/scripts/excel2csv.py
Comment (elysia-wang) [inline models/outside-cattle/scripts/excel2csv.py:1]: don't think we need to commit this file? seems like we just need the .csv it made

File: models/outside-cattle/scripts/main.py
Comment (elysia-wang) [inline models/outside-cattle/scripts/main.py:1]: might need to run `streamlit run main.py` from the scripts folder—it couldn't find the .csv

File: models/outside-cattle/scripts/main.py [hunk 1]
```diff
@@ -1,21 +1,157 @@
...
+       'Protozoa: dasytricha', 'Protozoa: diplodinium', 'Protozoa: diploplastron', 'Protozoa: entodinium', 'Fungi: neocallimastix', 'Fungi: piromyces', 'Fungi: caecomyces', 'Fungi: aklioshbomyces', 'Herd IT1', 'Herd IT2', 'Herd IT3', 'Herd UK1', 'Herd UK2']
 
+methane_data = pd.read_csv('wallace_metadata.csv')
+methane_label = methane_data.iloc[:, -1]
+methane_data = methane_data.iloc[:, 1:-1]
+herd_data = methane_data.iloc[:,5:10]
+
+herd_model = joblib.load('knn_model_herd.pkl')
```
Comment (elysia-wang) [inline models/outside-cattle/scripts/main.py:31]: this led to a key error for me /:
KeyError: 118

**Review by elysia-wang — COMMENTED** [review]
we should also clean up some of the other stuff in the repo that we don't need anymore

**Review by aaren-aras — APPROVED** [review]
Jeeeeeeeeeeeeeeeeeez

**Pull request merged**